### ISSUE TITLE
Message delivered verification

### ISSUE THREAD

**Issue by deavmi** [Nutomic/ensichat#22]
Maybe it would be nice to have message deliverance notifications like in whatsapp. Get the protocol to send a message to the sender telling him that the message has been sent.

**Comment by deavmi** [comment]
Was about to say. This could help determine weather the phoen should stirebthe messages locally until the node comes online again.

**Comment by Nutomic** [comment]
This is a bit more complicated because messages can always be lost. So it could be that a message arrived, but the confirmation gets lost. And you have no way to know about that. And retrying forever also doesn't seem like a good idea.

**Comment by deavmi** [comment]
But surely you could say the same for the message getting lost.

**Comment by Nutomic** [comment]
That's right, which makes this even more complicated. We might want to wait until #26 is fixed.

**Comment by deavmi** [comment]
Okay. Also is a dynamic routing protocol planned?

**Comment by Nutomic** [comment]
This is now implemented in the protocol, but not displayed in the app yet.

**Comment by deavmi** [comment]
Awesome work man @Nutomic
